src/configLoaders/DefaultConfigLoader.ts

```diff
@@ -0,0 +1,29 @@
+import ConfigLoader from './ConfigLoader';
+import * as path from 'path';
+import JestConfiguration from './JestConfiguration';
+
+/**
+ * The Default config loader will load the Jest configuration using the package.json in the package root
+ */
+export default class DefaultConfigEditor implements ConfigLoader {
+  private _fs: any;
+  private _projectRoot: string;
+
+  constructor(projectRoot: string, fs: any) {
+    this._projectRoot = projectRoot;
+    this._fs = fs;
+  }
+
+  public loadConfig(): JestConfiguration {
+    const packageJson = this._fs.readFileSync(path.join(this._projectRoot, 'package.json'), 'utf8');
+
+    // Parse the package.json and return the Jest property
+    const jestConfig = JSON.parse(packageJson).jest;
```

Comment:
Does this mean we will only support jest config in 'package.json'? Not a separate "jest.config.js" file like explained here: https://facebook.github.io/jest/docs/en/configuration.html

Seems fairly strait forward to support that as well.

Reply:
Well, you can assign the config object manually in the stryker config
```javascript
jest: {
  config: require('path/to/jestConfig.js')
}
```

Reply:
Good call. Can we add it as an example in the readme?

Reply:
I think we can :+1: